ivy/functional/frontends/mindspore/ops/function/nn_func.py

```diff
@@ -372,6 +383,22 @@ def avg_pool2d(
     )
 
 
+@to_ivy_arrays_and_back
```

Comment:
Could you please use the following script to determine the data-types supported by this function and add the supported or unsupported dtypes to `with_supported_dtypes` or `with_unsupported_dtypes` decorator please?
```python
import mindspore.ops as ops
import mindspore.common.dtype as mstype

def check_bias_add_supported_dtypes():
    supported_dtypes = [
        mstype.float16, mstype.float32, mstype.float64,
        mstype.int8, mstype.int16, mstype.int32, mstype.int64,
        mstype.uint8, mstype.uint16
    ]
    unsupported_dtypes = []

    for dtype in supported_dtypes:
        try:
            _ = ops.bias_add([1, 2, 3], [0.1, 0.2, 0.3], data_type=dtype)
        except ValueError:
            unsupported_dtypes.append(dtype)

    return supported_dtypes, unsupported_dtypes

if __name__ == "__main__":
    supported_dtypes, unsupported_dtypes = check_bias_add_supported_dtypes()
    
    print("Supported Data Types:")
    for dtype in supported_dtypes:
        print(dtype)
    
    print("\nUnsupported Data Types:")
    for dtype in unsupported_dtypes:
        print(dtype)
```

Reply:
@hello-fri-end i really dont knew how to run that script,i really dont knew where should i add for getting those suggestions..can u pls tell me how to run that?